src/server/lib/latest-report.js

```diff
@@ -0,0 +1,46 @@
+/* eslint camelcase: 0 */
```

Comment:
can probably kill this line if you rename `period_id` to `periodId`

Reply:
Yeah, I just decided to leave it alone for consistency with other usages through the codebase.  🤷‍♂️